lib/mixins/property-effects.js

```diff
@@ -1472,6 +1493,9 @@ export const PropertyEffects = dedupingMixin(superClass => {
           this.__dataToNotify = this.__dataToNotify || {};
           this.__dataToNotify[property] = shouldNotify;
         }
+        if (legacyNoBatch) {
+          this._invalidateProperties();
```

Comment:
Add `return false` here since no extra work is needed after this.

Reply:
Done